install_config/install/deploy_router.adoc

```diff
@@ -498,6 +561,74 @@ $ modshard 3 \
     ROUTE_LABELS='sla=low'
 ----
 
+[[using-namespace-router-shards]]
+=== Using Namespace Router Shards
+
+Each
+link:../../architecture/core_concepts/projects_and_users.html#projects[project has its own namespace].
+The routes for a project can be handled by a selected router by using
+`NAMESPACE_LABELS`.  The router is given a selector for a `NAMESPACE_LABELS`
+label and the project that wants to use the router applies the `NAMESPACE_LABELS`
+label to its namespace.
```

Comment:
This is not very clear - does this mean sharding a single project or sharding across projects? You'd use `NAMESPACE_LABELS` to filter out routes (from multiple projects/namespaces) you want a router to service. And so for `"sharding"` across projects that works with `NAMESPACE_LABELS`. 
Within the scope of a single project however, `NAMESPACE_LABELS` makes no sense - you'd use `ROUTE_LABELS` to filter routes within that project/namespace.